### PR TITLE
Add SeatConstraintsExtension to extensions.proto

### PR DESCRIPTION
**Pull request by gil-cukierman**

### CODE REVIEW

File: beeswax/openrtb/seat_constraints.proto [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+syntax="proto2";
+package openrtb;
+
+option java_package = "com.beeswax.openrtb";
+option cc_enable_arenas = true;
+
+// Next available tag: 2
+message SeatConstraints {
+  optional int64 bid_floor_micros = 1 [default = 0];
```
Comment (hostdp6) [inline beeswax/openrtb/seat_constraints.proto]: maybe I'm confused about how protos work, but shouldn't there be an account id in here somewhere?

Comment (gil-cukierman) [reply beeswax/openrtb/seat_constraints.proto]: @hostdp6 account id is the key to the map https://github.com/BeeswaxIO/beeswax-api/blob/bf88c2cbaed139ae4c5be0ff7d3ead215b3586d9/beeswax/openrtb/extension.proto#L463

The extension `seat_constraints` containts:
1. A `default_seat_constraint`of type `SeatConstraint` and a map from `account_id` to `SeatConstraint`

Comment (hostdp6) [reply beeswax/openrtb/seat_constraints.proto]: i see, I guess I was expecting to see it explicitly.

**Comment by skywalkermml** [comment]
LGTM

**Comment by gil-cukierman** [comment]
@hostdp6 @skywalkermml any other concerns or comments on this one?

**Comment by hostdp6** [comment]
LGTM

**Pull request merged**